config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  
+  default_url_options host: "example.com"
```

Comment:
Any reasons why we're setting this here?

If you need this I'd recommend you set the host value via an Environment variable since this value will be different for each environments the app is deployed to